packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -66,10 +66,10 @@ export function isHostClientMobile(): boolean {
 export function isApiSupportedOnMobile(requiredVersion: string = defaultSDKVersionForCompatCheck): SdkError {
```

Comment:
Can we please change the signature to not return SdkError?

Reply:
yes, i think the only type i can add is a `never` type

Reply:
void?

Reply:
I've really been racking my brain. Wracking my brain? Trying to figure out how to fix this function. It's very small but it's also very misleading. Here is the best idea that I have come up with:

1) Change name to isMobileApiSupported
2) Change return value to boolean
3) Change line 74 to return true

I *think* the changed name would more closely match what the function is actually doing (validating that you are currently running on a mobile platform AND that the call you're trying to make will work on the current version  of the SDK the user is using), AND it makes the function named like it is returning a boolean return a boolean.

I still don't love that it only returns true OR throws, but I think this at least will make it a little better. Open to other suggestions though